src/classes/BDI_DataImport_API.cls

```diff
@@ -49,6 +49,15 @@ global with sharing class BDI_DataImport_API {
     global static final string RequireBestMatch = 'Best Match'; // Use the best match. Show error if no match.
     global static final string BestMatchOrCreate = 'Best Match or Create'; // Use the best match. Create opportunity if no match.
 
+    /*******************************************************************************************************
+    * @description master values for the BDI Status field picklist
```

Comment:
Does the change to using these instead of labels mean that the labels are deprecated and can be removed? Or, if they aren't protected, that a reference should be added to the UTIL_Namespace_TEST class?

Reply:
I removed the labels that had not yet been packaged.  I did forget to add a reference in our test class for the ones that must stay.  good catch.  i'll add that now and update the pull request.  thanks.